examples/trainercontroller_configs/epoch-level-eval-loss-patience.yaml

```diff
@@ -0,0 +1,14 @@
+controller-metrics:
+  - name: eval_loss_window
+    class: HistoryBasedMetric
+    arguments:
+      window-size: 2
+controllers:
+  - name: epoch-level-eval-loss-patience
+    triggers:
+      - on_epoch_end
+    rule: len(eval_loss_window["metrics"]) > 0 and eval_loss_window["metrics"][-1]["eval_loss"] > 2
+    patience:
+      patience-threshold: 2
```

Comment:
Is `patience_threshold` the right one or is `patience-threshold` the right one?

Reply:
addressed.